modules/EvManager/main/car_simulatorImpl.cpp

```diff
@@ -23,6 +24,19 @@ void car_simulatorImpl::ready() {
 
     setup_ev_parameters();
 
+    if (mod->config.keep_cross_boot_plugin_state) {
+        if (!mod->r_kvs.empty()) {
+            auto variant = mod->r_kvs.at(0)->call_load(simulated_already_plugged_in_key);
+            bool* value = std::get_if<bool>(&variant);
+            if (value != nullptr && *value) {
+                enabled = true;
+                std::string plugin_command{"iec_wait_pwr_ready;sleep 60"};
```

Comment:
With this `plug_in_command`, is another replug needed to actually start a charging session after a reboot?